src/adapter/src/coord/timeline.rs

```diff
@@ -193,15 +191,11 @@ impl Coordinator {
         timeline: &'a Timeline,
         initially: Timestamp,
         now: NowFn,
-        timestamp_oracle_impl: TimestampOracleImpl,
         pg_oracle_config: Option<PostgresTimestampOracleConfig>,
         global_timelines: &'a mut BTreeMap<Timeline, TimelineState<Timestamp>>,
     ) -> &'a mut TimelineState<Timestamp> {
         if !global_timelines.contains_key(timeline) {
-            info!(
-                "opening a new {:?} TimestampOracle for timeline {:?}",
-                timestamp_oracle_impl, timeline,
-            );
+            info!("opening a new TimestampOracle for timeline {:?}", timeline,);
```

Comment:
maybe leave something like "opening new CRDB/postgres TimestampOracle ...", just for clarity?

Reply:
Good idea, done.